src/EditorFeatures/Core/EditorFeatures.csproj

```diff
@@ -266,6 +266,15 @@
     <Compile Include="Implementation\Diagnostics\DiagnosticsSuggestionTaggerProvider.cs" />
     <Compile Include="Implementation\Diagnostics\SuggestionAdornmentManagerProvider.cs" />
     <Compile Include="Implementation\Diagnostics\SuggestionTag.cs" />
+    <Compile Include="Implementation\FindReferences\DefinitionItem.cs" />
```

Comment:
It's a nit, but I wonder how many of these actually have an editor dependency, and which could be moved down to features to be used from OmniSharp or XamarinStudio.